### ISSUE TITLE
The FM range could be much higher

### ISSUE THREAD

**Issue by MaZe6K** [quamplex/geonkick#27]
Hi there!

I think the range of the FM could be much higher, right now it's basically just a subtle warming up of oscillator 2, I think it would be much nicer if it reached through clonky all the way into harsh metally territory. Something like 1000-5000 Hz, or 48 semitones or something like that, to make real FM drum sounds.

**Comment by iurienistor** [comment]
@MaZe6K Yes, I need to correct a little the mathematics behind FM synthesis and recalculate the modulation index (beta index) range. Also, there is plan to add and envelope for the modulation index too, even though I don't know if this will be useful.

**Comment by MaZe6K** [comment]
Well the way it currently works the amplitude envelope of osc 1 serves that purpose, so I don't think another envelope woud be neccessary. But a much higher range would be useful.
